src/service/viewport-impl.js

```diff
@@ -1825,10 +1825,11 @@ function createViewport(ampdoc) {
   let binding;
   if (ampdoc.isSingleDoc() &&
       getViewportType(ampdoc.win, viewer) == ViewportType.NATURAL_IOS_EMBED) {
+    const iosMajorVersion = platformFor(ampdoc.win).getIosMajorVersion();
     if (isExperimentOn(ampdoc.win, 'ios-embed-wrapper')
         // The overriding of document.body fails in iOS7.
         // Also, iOS8 sometimes freezes scrolling.
-        && platformFor(ampdoc.win).getIosMajorVersion() > 8) {
+        && iosMajorVersion && iosMajorVersion > 8) {
```

Comment:
No need. Please revert this line. `null > 8` yields `false`, which is what we want.